### PR TITLE
[openstack_single] Added compute01 and 02.

### PR DESCRIPTION
**Pull request by daneel23**

### CODE REVIEW

File: classes/system/reclass/storage/system/openstack_single.yml [hunk 1]
```diff
@@ -34,3 +34,27 @@ parameters:
...
             opencontrail_compute_iface: eth0
+        compute01.openstack.local:
+          name: compute01
+          domain: openstack.local
+          classes:
+          - system.linux.system.ubuntu
+          - system.openstack.compute.single
+          params:
+            salt_master_host: ${_param:reclass_config_master}
+            single_address: 10.10.10.202
```
Comment (pupapaik) [inline classes/system/reclass/storage/system/openstack_single.yml:45]: 202 is mistake or you want to have two nodes with same ip for different heat templates?

File: classes/system/reclass/storage/system/openstack_single.yml [hunk 2]
```diff
@@ -34,3 +34,27 @@ parameters:
...
+            opencontrail_compute_iface: eth0
+        compute02.openstack.local:
+          name: compute02
+          domain: openstack.local
+          classes:
+          - system.linux.system.ubuntu
+          - system.openstack.compute.single
+          params:
+            salt_master_host: ${_param:reclass_config_master}
+            single_address: 10.10.10.204
```
Comment (pupapaik) [inline classes/system/reclass/storage/system/openstack_single.yml:57]: .203 or .204?

File: classes/system/reclass/storage/system/openstack_single.yml (hunk 1 above)
Comment (daneel23) [reply classes/system/reclass/storage/system/openstack_single.yml:45]: .202 is my intention -- to be used either for single-1-compute or single-2-computes variant. No conflict expected in such use.

**Comment by pupapaik** [comment]
LGTM

File: classes/system/reclass/storage/system/openstack_single.yml (hunk 2 above)
Comment (daneel23) [reply classes/system/reclass/storage/system/openstack_single.yml:57]: Was thinking about .203, would be nicer, but didn't want to involve anybody already using this model (and their documentation).

**Pull request merged**